dashboard/app/controllers/reference_guides_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ReferenceGuidesController < ApplicationController
+  include CurriculumHelper
+  authorize_resource
```

Comment:
can we try to use `load_and_authorize_resource` here? Also on that we update ability.rb for Reference Guides?